ci_support/run_docker_build.sh

```diff
@@ -37,8 +37,6 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
Effectively means you need to be building a recipe as root, which isn't great in general. We will have to come up with a better solution eventually.